packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -1190,9 +1190,8 @@ export class Vector3 {
      * Creates a vector normal (perpendicular) to the current Vector3 and stores the result in the given vector
      * @param result defines the Vector3 object where to store the resultant normal
      * returns the result
-     * @hidden
      */
-    public _getNormalToRef(result: DeepImmutable<Vector3>): Vector3 {
+    public getNormalToRef(result: DeepImmutable<Vector3>): Vector3 {

```

Comment:
Did you mean to make this one a public and not hidden?

Reply:
> Did you mean to make this one a public and not hidden?

Yes, I found it useful in doing some quaternion examples and I can think of other areas when it would prove useful so it makes sense to make it public. I will produce an example of use when I have finished dealing with the quaternions.

Reply:
The function requires more documentation as it does not explain how the normal is chosen but there is an infinity of possibilities ???

Reply:
@sebavan done